ledger/participant-integration-api/src/test/lib/scala/platform/store/migration/tests/MigrationEtqTests.scala

```diff
@@ -0,0 +1,597 @@
+          ),
+          newCreateEvent_125(
+            transaction_id = "txId502",
+            contract_id = "cId502",
+            event_offset = "eventOffset502",
+            event_sequential_id = 502L,
+            flat_event_witnesses = Vector.empty,
+            tree_event_witnesses = Vector.empty,
+            template_id = 0,
+          ),
+        )
+    )
+    migrateTo(dstMigration)
+    fetchTable(
+      Schema126.idFilterCreateStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterCreateNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterNonConsumingInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingStakeholder
+    ) shouldBe empty
+    fetchTable(
+      Schema126.idFilterConsumingNonStakeholderInformee
+    ) shouldBe empty
+    fetchTable(
+      Schema126.transactionMeta
+    ) shouldBe Seq(
+      row(
+        "transaction_id" -> "txId501",
+        "event_offset" -> "eventOffset501",
+        "event_sequential_id_first" -> 501L,
+        "event_sequential_id_last" -> 501L,
+      ),
+      row(
+        "transaction_id" -> "txId502",
+        "event_offset" -> "eventOffset502",
+        "event_sequential_id_first" -> 502L,
+        "event_sequential_id_last" -> 502L,
+      ),
+    )
+  }
+
+  private val dbDataTypes = new DbDataTypes(dbType)
+  object Schema125 {
+    import com.daml.platform.store.migration.MigrationTestSupport._
+    import dbDataTypes._
+
+    private val commonEventsColumns: List[(String, DbDataType)] = List(
+      "event_sequential_id" -> BigInt,
+      "workflow_id" -> Str.optional,
+      "command_id" -> Str.optional,
+      "application_id" -> Str.optional,
+      "submitters" -> StringArray.optional,
+      "contract_id" -> Str,
+      "tree_event_witnesses" -> StringArray,
+    )
+
+    private val commonTransactionEventsColumns: List[(String, DbDataType)] =
+      commonEventsColumns ::: List[(String, DbDataType)](
+        "ledger_effective_time" -> BigInt,
+        "event_offset" -> Str,
+        "node_index" -> Integer,
+        "transaction_id" -> Str,
+        "event_id" -> Str,
+        "template_id" -> Str,
+        "flat_event_witnesses" -> StringArray,
+      )
+
+    private val commonExerciseEventsColumns: List[(String, DbDataType)] =
+      commonTransactionEventsColumns ::: List(
+        "exercise_argument" -> Bytea,
+        "exercise_argument_compression" -> Integer.optional,
+        "exercise_result" -> Bytea.optional,
+        "exercise_result_compression" -> Integer.optional,
+        "create_key_value" -> Bytea.optional,
+        "create_key_value_compression" -> Integer.optional,
+        "exercise_choice" -> Str,
+        "exercise_actors" -> StringArray,
+        "exercise_child_event_ids" -> StringArray,
+      )
+
+    val divulgenceEvents: TableSchema =
+      TableSchema("participant_events_divulgence", "event_sequential_id")(commonEventsColumns: _*)
+        .++(
```

Comment:
Just curious, what's behind such stylistic choice to have a column in common set but then overriding it in this inline (e.g. submitters) and having two augmentation lists with contradicting entries (e.g. template_id)

Reply:
It's on oversight, fixing

Reply:
"submitters" is still defined twice:
```
private val commonEventsColumns: List[(String, DbDataType)] = List(
    ...   
    "submitters" -> StringArray.optional,
```
and then
```
TableSchema("participant_events_divulgence", "event_sequential_id")(commonEventsColumns: _*)
        .++(
          "event_offset" -> Str.optional,
          "create_argument" -> Bytea.optional,
          "create_argument_compression" -> Integer.optional,
          "submitters" -> IntArray.optional,
```
The same is repeated in each of the table schemes

Reply:
Fixed now